.github/workflows/github-publish.yaml

```diff
@@ -46,20 +46,6 @@ jobs:
         with:
           arguments: :${{ inputs.module }}:build
 
-      - name: Create Next Version Commit
-        run: |
-          git config --global user.name 'Responsive Tools'
-          git config --global user.email 'tools@responsive.dev'
-          ./gradlew :${{ inputs.module }}:markNextVersion \
-            -Prelease.incrementer=${{ inputs.increment }} \
-            -Prelease.dryRun | grep "Creating" | git commit --allow-empty -F -
-          git push
-
-      - name: Increment Version
-        uses: gradle/gradle-build-action@v2
-        with:
-          arguments: :${{ inputs.module }}:markNextVersion -Prelease.incrementer=${{ inputs.increment }}
-
       - name: Release
```

Comment:
we now _first_ run the release plugin to tag HEAD with the current version (what we've been building SNAPSHOTs of). Then we publish that and then create the next version. Note that this should be idempotent if publish fails. The next iteration of the job will just skip the release step with ("nothing to release")

Reply:
makes sense, previously we were working on 0.X.0 and releasing 0.X+1.0, this swaps that 👍